homework36/src/components/ReactionsCounters.tsx

```diff
@@ -0,0 +1,36 @@
+import { Component } from "react";
+
+class ReactionsCounters extends Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            emojis: this.props.emojis,
+        };
+    }
+
+    vote(i) {
+        let newEmojiVote = this.state.emojis;
```

Comment:
```suggestion
        let newEmojiVote = [...this.state.emojis];
```

не слід змінювати стейт напряму, краще створюйте копію